packages/frontend/src/components/MkNoteDetailed.vue

```diff
@@ -482,6 +484,16 @@ async function clip() {
 	os.popupMenu(await getNoteClipMenu({ note: note.value, isDeleted }), clipButton.value).then(focus);
 }
 
+async function reactionMuteToggle(emojiName: string) {
+	if (!mutedReactions.value.includes(emojiName)) {
+		mutedReactions.value.push(emojiName);
+		defaultStore.set('mutedReactions', mutedReactions.value);
+	} else {
+		mutedReactions.value.splice(mutedReactions.value.indexOf(emojiName), 1);
```

Comment:
https://github.com/MisskeyIO/misskey/blob/f9eb205dc8512c64200cda3da4e43958f1ba2d31/packages/frontend/src/pages/settings/emoji-picker.vue#L197
ピン留めリアクションがこれなので、こんな感じにするのが良さそう
（もしかしたらコード間違ってるかもしれん）
```suggestion
		mutedReactions.value = mutedReactions.value.filter(x => x !== emojiName)
```